Formula/icu4c.rb

```diff
@@ -8,8 +8,10 @@ class Icu4c < Formula
 
   livecheck do
     url :stable
-    strategy :github_latest
     regex(%r{href=.*?/tag/release[._-]v?(\d+(?:[.-]\d+)+)["' >]}i)
```

Comment:
```suggestion
    regex(/^release[._-]v?(\d+(?:[.-]\d+)+)$/i)
```

Oh, interesting. It looks like GitHub allows you to provide suggestions for lines that haven't been modified in a PR now. This has been a pain point for me for a good while, so I'm glad to see it's been addressed.

I mentioned this regex change in the strategy comment but it was easy to overlook. Since we switched this `livecheck` block to the Git strategy, we need to update it to use a contextually-appropriate regex. `GithubLatest` works with an HTML page but `Git` matches against tag strings.

Reply:
Sorry for I forgot to update the regular expression.